_sections/building.html

```diff
@@ -19,15 +19,15 @@
     _slug: start-building-your-own-innovative-applications
     title: Start building your own innovative applications
     show_text: true
-    body: "<p>We're not just a magic idea whitepaper. You can build cutting edge distributed apps in JavaScript or Lisp on our platform right now! Your apps will have access to core services like: a customizable UI, the Holochain DPKI app that manages keys and identities across holochain applications,
+    body: "<p>We're not just a magic idea whitepaper. You can build cutting edge distributed apps in Rust on our platform right now! Your apps will have access to core services like: a customizable UI, the Holochain DPKI app that manages keys and identities across holochain applications,
```

Comment:
Well to be fair, javascript still applies, right? You'll be able to use webassembly to connect to the javascript world.